### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by sshanks-kx**
removed repeating license instructions
details on on-demand and commercial license repeated moved relevent sections into appropriate place (e.g. license server moved under on-demand)

### CODE REVIEW

File: docs/interfaces/q-server-for-odbc3.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ description: The ODBC3 server allows applications to query kdb+ via the ODBC int
 # :fontawesome-solid-database: kdb+ server for ODBC3
 
 
-
-The ODBC3 server allows applications to query kdb+ via the ODBC interface.  
+The ODBC3 driver allows applications to query kdb+ via the ODBC interface.
```
Comment (vcalescu) [inline docs/interfaces/q-server-for-odbc3.md]:
```suggestion
The ODBC3 driver allows applications to query kdb+ through the ODBC interface.  
```

File: docs/interfaces/q-server-for-odbc3.md [hunk 2]
```diff
@@ -15,25 +14,52 @@ Requirements: V3.2 or later.
...
+q)\l ps.k
+```
+
+Run `.s.ver` to check that ps.k has been loaded and for the current version e.g.
+```q
+q).s.ver
+1.15
+```
+
+The following is an example of staring kdb+ with ps.k listening on port 5000
```
Comment (vcalescu) [inline docs/interfaces/q-server-for-odbc3.md]:
```suggestion
The following is an example of starting kdb+ with ps.k listening on port 5000
```

File: docs/interfaces/q-server-for-odbc3.md [hunk 3]
```diff
@@ -15,25 +14,52 @@ Requirements: V3.2 or later.
...
+
+```bash
+q ps.k -p 5000
+```
+
+### ODBC driver and DSN configuration
+
+Software that uses ODBC connections refer to individual connections via a `DSN`(Data Source Name).
```
Comment (vcalescu) [inline docs/interfaces/q-server-for-odbc3.md]:
```suggestion
Software that uses ODBC connections refer to individual connections using a `DSN`(Data Source Name). 
```

File: docs/interfaces/q-server-for-odbc3.md [hunk 4]
```diff
@@ -44,36 +70,68 @@ $ cd qodbc3/l64
...
+DRIVERS............: /etc/odbcinst.ini
+SYSTEM DATA SOURCES: /etc/odbc.ini
+FILE DATA SOURCES..: /etc/ODBCDataSources
+USER DATA SOURCES..: /root/.odbc.ini
+SQLULEN Size.......: 8
+SQLLEN Size........: 8
+SQLSETPOSIROW Size.: 8
+```
+
+Add the driver to your list of drivers by editing the driver ini file ( `/etc/odbcinst.ini`). An example of the definition follows (note: the driver locaton should be altered to the location choosen on your system):
```
Comment (vcalescu) [inline docs/interfaces/q-server-for-odbc3.md]:
```suggestion
Add the driver to your list of drivers by editing the driver `ini` file ( `/etc/odbcinst.ini`). An example of the definition follows (note: the driver location should be altered to the location chosen on your system):
```

File: docs/interfaces/q-server-for-odbc3.md [hunk 5]
```diff
@@ -97,3 +152,69 @@ The driver translates SQL expressions into q and inherits q’s data model. This
...
 Also, SQL selects from partitioned tables are not supported – one should pre-select from a partitioned table using the `q()` function instead.
 
+`test.q` provides additional examples of SQL usage, including the create/insert/update/delete statement syntax.
+
+## Debugging
+
+ODBC implementations provide a tracing capability to log interactions with an ODBC driver. This can aid in diagnosing any issues. Tracing can have a detremental 
+effect to ODBC performance.
+
+This is a feature of the ODBC system, and not a feature unique to the kdb+ driver. Associated documentation, features and troubleshooting can be found online for your OS.
```
Comment (vcalescu) [inline docs/interfaces/q-server-for-odbc3.md]:
```suggestion
This is a feature of the ODBC system, and not a feature unique to the kdb+ driver. Associated documentation, features, and troubleshooting can be found online for your OS.
```

File: docs/interfaces/q-server-for-odbc3.md [hunk 6]
```diff
@@ -97,3 +152,69 @@ The driver translates SQL expressions into q and inherits q’s data model. This
...
+
+!!! warn "Connection details can be logged, therefore the log file should be stored in a private location"
+
+### Windows
+
+In the ODBC data source administrator, click _Start Tracing_ on the _Tracing_ tab. 
+
+An example of the data recorded to the `SQL.LOG`:
```
Comment (vcalescu) [inline docs/interfaces/q-server-for-odbc3.md]:
```suggestion
See below an example of the data recorded to the `SQL.LOG`:
```

File: docs/interfaces/q-server-for-odbc3.md [hunk 7]
```diff
@@ -97,3 +152,69 @@ The driver translates SQL expressions into q and inherits q’s data model. This
...
+		        PTR                 0x00000256A79BB1D0 [       2] "1"
+		        SQLLEN                  4092
+		        SQLLEN *            0x000000DAA198E160 (2)
+```
+
+Please ensure tracing is disabled after debugging complete.
+
+### Linux
+
+In the `[ODBC]` section of the driver ini file (`/etc/odbcinst.ini`), set `Trace` to `1` to enable tracing and `TraceFile` to the location of the log file to create.
```
Comment (vcalescu) [inline docs/interfaces/q-server-for-odbc3.md]:
```suggestion
In the `[ODBC]` section of the driver `ini` file (`/etc/odbcinst.ini`), set `Trace` to `1` to enable tracing and `TraceFile` to the location of the log file to create. 
```

File: docs/wp/data-visualization/index.md [hunk 8]
```diff
@@ -254,6 +209,61 @@ option when placed in a view. However, there are ways to make them more
 dynamic and flexible. This will be explored below in
 _Dynamic Parameters_.
 
+#### Dynamic parameters
+
+As mentioned above in _Simple parameters_, Tableau
+parameters are limited to static values, and a single select option when
+placed in a view. However, there are a number of ways to make parameters
```
Comment (vcalescu) [inline docs/wp/data-visualization/index.md]:
```suggestion
placed in a view. However, there are several ways to make parameters
```

File: docs/wp/data-visualization/index.md [hunk 9]
```diff
@@ -254,6 +209,61 @@ option when placed in a view. However, there are ways to make them more
...
+##### Parameters with calculated fields
+
+Using parameters in conjunction with Tableau’s calculated-field
+functionality can be a convenient and flexible tool in calculations as
+well as graphical representation. This is useful when the output the
+user wants to see is dependent on an input parameter, and a field needs
+to be adjusted accordingly.
+
+For example, in the user-defined `Calculation1` logic below, the quantity
+field will be divided by a different amount depending on the chosen
```
Comment (vcalescu) [inline docs/wp/data-visualization/index.md]:
```suggestion
field is divided by a different amount depending on the chosen
```

File: docs/wp/data-visualization/index.md [hunk 10]
```diff
@@ -254,6 +209,61 @@ option when placed in a view. However, there are ways to make them more
...
+Category value. 
+
+![](img/image12.png)
+
+Below is sample output from when the user selects
+a _Category_ value of `EQ`.
+
+![](img/image13.png)
+
+In contrast, when the user selects `CORP` the calculated field will be
```
Comment (vcalescu) [inline docs/wp/data-visualization/index.md]:
```suggestion
In contrast, when the user selects `CORP` the calculated field is
```

File: docs/wp/data-visualization/index.md [hunk 11]
```diff
@@ -498,18 +453,24 @@ on the general publishing procedure, which involves publishing from within an al
...
-connections, it is recommended to use the `tabcmd` command-line utility
-which, among other things, enables the user to publish to Tableau Server
-from the command line. This utility allows the user to deploy sheets
+This process is manual and prone to errors. 
+
+### Publishing using tabcmd
+
+For kdb+
+connections, it is recommended to use the [`tabcmd`](https://help.tableau.com/current/server/en-us/tabcmd.htm) command-line utility
+which, among other things, enables the user to [publish to Tableau Server
```
Comment (vcalescu) [inline docs/wp/data-visualization/index.md]:
```suggestion
which, among other things, enables you to [publish to Tableau Server
```

File: docs/wp/data-visualization/index.md [hunk 12]
```diff
@@ -498,18 +453,24 @@ on the general publishing procedure, which involves publishing from within an al
...
+from the command line](https://help.tableau.com/current/server/en-us/tabcmd_cmd.htm#iddf805b62-18ff-4497-9245-adc6905b2084). This utility allows the user to deploy sheets
```
Comment (vcalescu) [inline docs/wp/data-visualization/index.md]:
```suggestion
from the command line](https://help.tableau.com/current/server/en-us/tabcmd_cmd.htm#iddf805b62-18ff-4497-9245-adc6905b2084). This utility allows you to deploy sheets
```

**Review by vcalescu — CHANGES_REQUESTED** [review]

**Comment by sshanks-kx** [comment]
changes made, thanks

**Review by vcalescu — APPROVED** [review]

**Pull request merged**